modules/process/SV/SomaticDellyCall.nf

```diff
@@ -30,8 +30,21 @@ process SomaticDellyCall {
     --outfile ${idTumor}__${idNormal}_${svType}.filter.bcf \
     ${idTumor}__${idNormal}_${svType}.bcf
 
+  # Filter variants that have low supporting reads in the tumor or high supporting reads in the normal
+  # DV = discordant reads
+  # RV = split reads
+  bcftools view \\
+    -s ${idTumor},${idNormal} \\
+    ${idTumor}__${idNormal}_${svType}.filter.bcf | \\
+  bcftools filter \\
+    --soft-filter tumor_read_supp -m + \\
+    -e "FORMAT/DV[0] < 5 | FORMAT/RV[0] < 2" | \\
+  bcftools filter \\
+    --soft-filter normal_read_supp -m + \\
+    -e "FORMAT/DV[1] > 0 | FORMAT/RV[1] > 0" | \\
+  bcftools view --output-type z > \\
+    ${idTumor}__${idNormal}_${svType}.delly.vcf.gz
```

Comment:
If we are calling manta output as `filtered`, maybe we want to be consistant here on delly output.

Reply:
done, they are now both `*.manta.vcf.gz` and `*.delly.vcf.gz`.